mc-sema/peToCFG/peToCFG.cpp

```diff
@@ -618,6 +632,20 @@ NativeModulePtr readProtoBuf(std::string fName, const llvm::Target *T) {
       externData.push_back(deserializeExtData(ed));
     }
 
+    for(int i = 0; i < serializedMod.offset_tables_size(); i++) {
+        const ::OffsetTable &ot = serializedMod.offset_tables(i);
+
+        std::vector< std::pair<VA,VA> > v;
+
+        for( int j = 0; j < ot.table_offsets_size(); j++) {
+            v.push_back( std::pair<VA,VA> ( ot.table_offsets(j), ot.destinations(j) ) );
```

Comment:
This is a bit funny to me. You build up a vector of pairs here, only to turn it into a map later on. Why not go straight to a map? Do you foresee handling multiple non-unique entries in the future to be necessary? Or is it somthing about the table having the same size in the emitted binary?